script/strip-binaries.py

```diff
@@ -22,7 +22,7 @@ def strip_binary(binary_path, target_cpu):
     strip = 'mips64el-redhat-linux-strip'
   else:
     strip = 'strip'
-  execute([strip, binary_path])
+  execute([strip, '-p', binary_path])
```

Comment:
I generally prefer to use long-form flags in scripts to reduce the mystery somewhat.
```suggestion
  execute([strip, '--preserve-dates', binary_path])
```